File: 01x10/c-006/main.js
function sumDifSquare(num) {
    const x = [...Array(num)]
        .map((_, i) => (i + 1) ** 2)
        .reduce((a, b) => a + b)
    const y = [...Array(num)]
        .map((_, i) => i + 1)
        .reduce((a, b) => a + b) ** 2
    return y - x
}

console.log(sumDifSquare(100))

File: 01x10/c-006/main.rs
fn sum_dif_square(num: i32) -> i32 {
    let x: i32 = (1..(num + 1))
        .map(|a| a.pow(2))
        .sum();
    let y: i32 = (1..(num + 1))
        .sum();
    return y.pow(2) - x;
}

fn main() {
    println!("{}", sum_dif_square(100));
}
